libs/utils/android/workloads/exoplayer.py

```diff
@@ -0,0 +1,184 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2017, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import re
+import os
+import logging
+
+from subprocess import Popen, PIPE
+
+from time import sleep
+
+from android import Screen, System, Workload
+
+# Regexps for benchmark synchronization
+
+REGEXPS = {
+    'start'    : '.*Displayed com.google.android.exoplayer2.demo/.PlayerActivity',
+    'duration' : '.*period \[(?P<duration>[0-9]+.*)\]',
+    'end'      : '.*state \[.+, .+, E\]'
+}
+
+class Exoplayer(Workload):
+    """
+    Android Exoplayer workload
```

Comment:
Might be useful to add a note explaining how to get hold of the APK?

Reply:
Ah yes, that is something I wanted to ask - what's the process for adding an APK to LISA ? It's all built from https://github.com/google/ExoPlayer (after some investigation with Chris we realized we could just use the default 'demo' application).